chrome/word-manager.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Handles the parsing of the page text as well as saving the words.
+ *
+ * @param {string} text
+ * @param {string} language
+ * @constructor
+ */
+WordManager = function(text, language) {
+  this.text_ = text;
+  this.language_ = language;
```

Comment:
Please convert language to the enum right away - there is no point in keeping original value